src/teleop/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -261,7 +261,7 @@ import _ from "lodash";
 import L from "leaflet";
 import ROSLIB from "roslib";
 
-let interval;
```

Comment:
should probably change the name of this to rover_stuck_interval since theres two of them now.

Reply:
done